frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/BaseRebaseBranchOntoParentAction.java

```diff
@@ -35,6 +35,7 @@
  * Expects DataKeys:
  * <ul>
  *  <li>{@link DataKeys#KEY_GIT_MACHETE_REPOSITORY}</li>
+ *  <li>{@link DataKeys#KEY_SELECTED_BRANCH_NAME}</li>
```

Comment:
Is it intended? `RebaseSelectedBranchOntoParentAction` does need `KEY_SELECTED_BRANCH_NAME`, but `RebaseCurrentBranchOntoParentAction` doesn't

Reply:
The question is whether the key annotated by inheriting class shall be annotated by the base class. My thinking was that generally, the answer is no but as we use a template method via `IBranchNameProvider::getNameOfBranchUnderAction`, annotating of any key used by its implementation makes sense. 

However, if you have a different opinion I am willing to revert these changes. It was cleaner before for sure.